docs/development/legacy/libraries/email.md

```diff
@@ -30,8 +30,9 @@ The Email class will automatically create all email headers and will process the
     ee()->load->library('email');
     ee()->load->helper('text');
 
-    ee()->email->wordwrap = true;
-    ee()->email->mailtype = 'text';
+    ee()->email->set_wordwrap(TRUE);
+    ee()->email->set_mailtype('html');
```

Comment:
Thanks @nep!  It looks like the old lines work fine with the library but I like your changes to using the library methods instead, sometimes they provide a little extra logic in setting these attributes 👍